packages/connect-thegraph/src/connector.ts

```diff
@@ -31,6 +31,22 @@ function getOrgSubgraphUrl(network: Network): string | null {
   return null
 }
 
+function appFiltersToQueryFilter(appFilters: AppFilters) {
+  const queryFilter = {} as any
+
+  if (appFilters.name) {
+    queryFilter.repoName_in = appFilters.name.map(name =>
```

Comment:
Maybe we should also filter the registry? One idea is that `queryFilter` be a tuple: `[name, registry]`

Reply:
Do you think that would be useful initially?

I’m thinking that it might be tricky to implement, as I couldn’t find a way to use a OR at the `where` level. This is why it is not possible right now to pass e.g. an address and a different app name.

If we also filter by registry, I think we would have the same kind of problem, where a list of names using different registries would be a problem as a query, e.g.

```tsx
const apps = await org.apps({
  appName: [
    ['voting', 'aragonpm.eth'],
    ['token-manager', 'somethingelse.eth']
  ]
})
```

I was thinking about having this as a base for now, and investigate together on the filters we want and maybe find a way to express an OR in the `where` expression.

Reply:
Make sense, as we discuss async let's think more about it next week. I agree this is good enough for now.